backend/substrapp/ledger_utils.py

```diff
@@ -267,37 +267,6 @@ def get_object_from_ledger(pk, query):
     return query_ledger(fcn=query, args={'key': pk})
 
 
-def _wait_until_status_after_timeout(tuple_type, tuple_key, expected_status):
-    query_fcns = {
-        'traintuple': 'queryTraintuple',
-        'testtuple': 'queryTesttuple',
-        'compositeTraintuple': 'queryCompositeTraintuple',
-        'aggregatetuple': 'queryAggregatetuple',
-    }
-    query_fcn = query_fcns[tuple_type]
-
-    max_tries = getattr(settings, 'LEDGER_MAX_RETRY_TIMEOUT', 5)
-    trie = 1
-    backoff = 5
-
-    while trie <= max_tries:
-        # sleep first as this is executed right after a request raising a timeout error
-        time.sleep(trie * backoff)
-
-        tuple_ = query_ledger(fcn=query_fcn, args={'key': tuple_key})
-        status = tuple_['status']
-        if status == expected_status:
-            return
-
-        logger.error(
-            f'{tuple_type} {tuple_key} wrong status {status}: expecting {expected_status} (trie {trie})'
-        )
-        trie += 1
-
-    raise LedgerTimeoutNotHandled(
```

Comment:
I think this exception is now unused and could be removed.

Reply:
good catch, removed here 93c430b